src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/QueryInterfaceTests.cs

```diff
@@ -124,15 +124,6 @@ public void QueryInterface_NoSuchInterface_Success(object o, string iidString)
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.AnyUnix)]
-        public void QueryInterface_Unix_ThrowsPlatformNotSupportedException()
```

Comment:
Is this an incorrectly resolved merge conflict? You may want to rebase, squash and force push to cleanup the git history.

Reply:
Thanks Jan! I had trouble rebasing to the origin :-(. Tried rebasing to the upstream and copying the changes and pushing,